src/Export/VTK.jl

```diff
@@ -17,9 +17,11 @@ nodes_to_vtkorder(cell::AbstractCell) = collect(cell.nodes)
 
 """
     vtk_grid(filename::AbstractString, grid::Grid)
+    vtk_grid(filename::AbstractString, dh::DofHandler)
```

Comment:
I think we are missing docs for the compress arg.

Reply:
Yes, I wasn't sure if that should be documented/public

One option would be to just use `kwargs...` and direct to `WriteVTK.jl`'s documentation, i.e https://juliavtk.github.io/WriteVTK.jl/stable/grids/syntax/#Data-formatting-options for those?
(Its default is also `compress=true`)